calico_cni_k8s_test.go

```diff
@@ -309,6 +309,93 @@ var _ = Describe("CalicoCni", func() {
 					}))
 				})
 			})
+
+			Context("using ipAddrs annotation to assign IP address to a pod, through Calico IPAM", func() {
```

Comment:
I tried adding those but our (dind?) test framework seems to have some issues with the netlink library and IPv6 when it tries to `DoNetworking` for IPv6 address